packages/phoenix-ng/projects/phoenix-ui-components/lib/components/ui-menu/object-clipping/object-clipping.component.ts

```diff
@@ -10,21 +10,27 @@ import { EventDisplayService } from '../../../services/event-display.service';
 })
 export class ObjectClippingComponent {
   clippingEnabled: boolean;
-  clippingAngle: number;
+  startClippingAngle: number;
+  openingAngle: number;
 
   constructor(private eventDisplay: EventDisplayService) {
     const stateManager = this.eventDisplay.getStateManager();
     stateManager.clippingEnabled.onUpdate(
       (clippingValue) => (this.clippingEnabled = clippingValue)
     );
-    stateManager.clippingAngle.onUpdate(
-      (value) => (this.clippingAngle = value)
+    stateManager.startClippingAngle.onUpdate(
+      (value) => (this.startClippingAngle = value)
     );
   }
 
-  changeClippingAngle(change: MatSliderChange) {
-    const angle = change.value;
-    this.eventDisplay.getUIManager().rotateClipping(angle);
+  changestartClippingAngle(change: MatSliderChange) {
```

Comment:
NIT: Rename to `changeStartClippingAngle`

Reply:
Sure, my bad.